tests/certification/configuration/redis/README.md

```diff
@@ -0,0 +1,17 @@
+# Redis Configuration Store certification testing
+
+This project aims to test the Redis Configuration Store component under various conditions.
+
+## Test plan
+
+## Test subscribe with different events:
+1. Subscribe to redis configuration store
```

Comment:
Are we not planning to test get and getAll herer ?

Reply:
It is already done as part of the conformance test.

Reply:
Thank you for not duplicating tests @shivamkm07. That's a big pet peeve of mine.

No basic tests handled by conformance in certification :)